packages/venia-ui/lib/components/CheckoutPage/checkoutPage.css

```diff
@@ -0,0 +1,112 @@
+.root {
+    padding: 2.5rem 3rem;
+    max-width: 1080px;
+    margin: 0 auto;
+}
+
+.body {
+    display: grid;
+    gap: 2rem;
+    grid-template-columns: 2fr 1fr;
+    /* The summary grid item spans the entire right column. */
+    grid-template-areas:
+        'shipping_information summary'
```

Comment:
Because we have named areas here for potentially hidden nodes there is some extra blank space created by the grid:

[![Image from Gyazo](https://i.gyazo.com/5bd4ae4c52c7e0da18830e3c7cd5d086.png)](https://gyazo.com/5bd4ae4c52c7e0da18830e3c7cd5d086)

I think you may be able to replace the names with `.`. The structure should be the same but it will compress the blank spaces.

Reply:
UI has changed. This is not relevant anymore.